gradle.properties

```diff
@@ -17,3 +17,5 @@
 org.curioswitch.curiostack.version = 0.0.183.1
 
 org.gradle.jvmargs=-Xmx2048m
+
+version = 0.3.0
```

Comment:
This looks reasonable but how does it interact with curiostack gradle plugin? IIRC, that also sets the version for the cloud build and also snapshots.